docs/src/developer/developer/pr-guidelines.md

```diff
@@ -81,7 +81,7 @@ Add to `charts/nginz/values.yaml`, under the `galley` section:
 If a PR adds new configuration options for say brig, the following files need to be edited:
 
 * [ ] The parser under `services/brig/src/Brig/Options.hs`
-* [ ] The integration test config: `services/brig/brig.integration.yaml`
+* [ ] The integration test config: `services/brig/brig.integration.yaml` and `charts/cargohold/templates/tests/configmap.yaml`
```

Comment:
ah, wait, this was already documented, but you have to read the context carefully.  this list is an example for the context where you made changes to brig.  for cargohold, most items in this list will change.

do you have a suggestion how to make this more clear?  maybe introduce variables $Service and $service to use in the list instead of `brig`?

Reply:
Variables would work, I think. I'll sit with this a bit and see what I can come up with.